DEVELOPING_TUTORIALS.md

```diff
@@ -388,6 +387,17 @@ You may (optionally) use [markdown formatting](https://guides.github.com/feature
 
 If this is the last lesson in your tutorial, please create a success message that notes that the user has completed the whole tutorial. For example, `Great job! You've completed this series of lessons!`)
```

Comment:
[](url)
```suggestion
If this is the last lesson in your tutorial, please create a success message that notes that the user has completed the whole tutorial. For example, `Great job! You've completed this series of lessons!`)

You can optionally use the [utils.validationMessages](#utilsvalidationmessages) utility to create commonly used validation messages. 
```

Reply:
I'd love to add in-line mentions for every common use case of each of the util functions so people find them in context.